ci/Dockerfile-envoy-image

```diff
@@ -2,5 +2,6 @@ FROM ubuntu:14.04
 
 ADD build/source/exe/envoy /usr/local/bin/envoy
 RUN apt-get update && apt-get install -y \
-    curl \
-    python-pip
+    build-essential
```

Comment:
Yeah just for strip. I was trying to find if it could be installed individually but didn't find an alt package for it.